commons-dlp/src/main/java/org/exoplatform/commons/dlp/processor/impl/DlpOperationProcessorImpl.java

```diff
@@ -86,14 +87,20 @@ public synchronized void process() {
       // Loop until the number of data retrieved from dlp queue is less than
       // BATCH_NUMBER (default = 1000)
       int totalProcessed = 0;
-      int processedOperations;
-      do {
-        LOGGER.debug("Process Bulk DLP operation (size : {})", batchNumber);
-        processedOperations = processBulk();
-        totalProcessed += processedOperations;
-        LOGGER.debug("{} DLP Operation processed, total processed : {}", processedOperations, totalProcessed);
-      } while (processedOperations >= batchNumber);
-      LOGGER.info("Dlp Operation Processor proceed {} queue elements by batches of {}", totalProcessed, BATCH_NUMBER_DEFAULT);
+      int offset = 0;
+      long total = dlpOperationDAO.count();
+      while (offset<total) {
+        LOGGER.debug("Process Bulk DLP operation (offset : {}, total {})", offset, total);
+        int stayingQueue=processBulk(offset);
+        int processedOperations=batchNumber-stayingQueue;
+        total=dlpOperationDAO.count();
+        offset+=stayingQueue;
+        totalProcessed=totalProcessed+processedOperations;
```

Comment:
Format the code please !

Reply:
Can you explain what is the problem in the format ?
Thanks